src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/MessageboxInstancesController.cs

```diff
@@ -48,8 +50,9 @@ public class MessageBoxInstancesController : ControllerBase
         [HttpGet("{instanceOwnerPartyId:int}")]
         public async Task<ActionResult> GetMessageBoxInstanceList(int instanceOwnerPartyId, [FromQuery] string state, [FromQuery] string language)
         {
-            string[] allowedStates = new string[] { "active", "archived", "deleted" };
-            string[] acceptedLanguages = new string[] { "en", "nb", "nn-no" };
+            string[] allowedStates = { "active", "archived", "deleted" };
+            string[] acceptedLanguages = { "en", "nb", "nn-no" };
```

Comment:
as the nre template is merged into master nn-no can be set to nn. Change has already been made in SBL.

Reply:
Corrected